src/clj/rems/api/schema.clj

```diff
@@ -43,7 +43,8 @@
    :round s/Num
    :event s/Str
    :comment (s/maybe s/Str)
-   :time DateTime})
+   :time DateTime
+   :eventdata s/Any})
```

Comment:
I'm not so sure about this...

Reply:
`{s/Keyword s/Any}` perhaps?

Reply:
I think an opaque s/Any is pretty clear. The contents of eventdata is up to the event implementations. I'm not sure whether we should even publish it via the api.